src/main/java/com/nike/cerberus/command/composite/RotateCertificatesCommand.java

```diff
@@ -16,11 +16,15 @@
 
 package com.nike.cerberus.command.composite;
 
+import com.beust.jcommander.Parameter;
 import com.beust.jcommander.Parameters;
 import com.nike.cerberus.command.Command;
+import com.nike.cerberus.command.validator.UploadCertFilesPathValidator;
 import com.nike.cerberus.operation.Operation;
 import com.nike.cerberus.operation.composite.RotateCertificatesOperation;
 
+import java.nio.file.Path;
```

Comment:
Were these imports already used but never imported?

Or are this a mistake?

Reply:
mistake.

- [ ] remove unused imports